### ISSUE TITLE
Outgoing ActivityPub message feature wrong context

### ISSUE THREAD

**Issue by MrPetovan** [owncast/owncast#2469]
Owncast's outgoing ActivityPub messages mention the `https://joinmastodon/ns` namespace, but the format is wrong.

Instead of 
```
{
    "@context": [
        "https://www.w3.org/ns/activitystreams",
        "http://joinmastodon.org/ns",
        "https://w3id.org/security/v1",
        ...
    ],
    ...
```
It should be
```
{
    "@context": [
        "https://www.w3.org/ns/activitystreams",
        {
            "toot": "http://joinmastodon.org/ns#"
        },
        "https://w3id.org/security/v1",
        ...
    ],
    ...
```
Doing otherwise triggers errors in JsonLD validation on receiving systems like Friendica. See https://github.com/friendica/friendica/issues/12419#issuecomment-1358381931

Thank you for your time!

**Comment by gabek** [comment]
Sounds like this is filed with go-fed already? https://github.com/go-fed/activity/issues/152. We use that for the serialization of ActivityPub objects.

**Comment by MrPetovan** [comment]
Indeed, I wasn't aware of the dependency, thank you for looking into it!

**Comment by tsmethurst** [comment]
@gabek We've just done some changes on our fork that implement `@context` inlining, feel free to borrow: https://github.com/superseriousbusiness/activity/pull/25
